src/js/components/Widgets/ReadMore.jsx

```diff
@@ -34,7 +34,7 @@ export default class ReadMore extends Component {
           link_text = "More";
         }
         if (collapse_text === undefined) {
-          collapse_text = " ...Less";
+          collapse_text = " ...Show Less  ";
```

Comment:
We added the "...Show Less" so we could play with it and see how it feels, and I have found that it really does help, especially in mobile view.